src/foam/java/refinements.js

```diff
@@ -834,6 +842,9 @@ foam.CLASS({
       var c = info.getMethod('comparePropertyToObject');
       c.body = 'return compareValues((String) key, get_(o));';
 
+      var sql = info.getMethod('getSQLType');
```

Comment:
This could be done with a property:
{
  name: 'sqlType',
  expression: function(width) {
    return 'VARCHAR(' + this.width + ')';
  }
}

Reply:
`this.width` or just `width`?